MLS.Agent/CommandLine/StartupOptions.cs

```diff
@@ -42,7 +42,8 @@ public static StartupOptions FromCommandLine(string commandLine)
             bool enablePreviewFeatures = false,
             string package = null,
             string packageVersion = null,
-            ParseResult parseResult = null)
+            ParseResult parseResult = null,
+            string port = null)
```

Comment:
>string [](start = 12, length = 6)

This should be int.

Reply:
If it is int then we will have to specify some value. What should that be ?

Reply:
Also, I am checking if this value or not to determine if we need to use this port or give a random port. I can make this as int and set the default value to 0 and then check that if options.Port is 0 then we should start on a random port. How does that sound?

Reply:
If you want to distinguish between whether it was specified or not, you could use `int?`.

Reply:
Done. Thanks!